FWCore/Services/plugins/InitRootHandlers.cc

```diff
@@ -771,6 +771,11 @@ namespace edm {
 
       if (not threadTracker_) {
         threadTracker_ = std::make_unique<ThreadTracker>();
+        iReg.watchPostEndJob([]() {
+          if (threadTracker_) {
```

Comment:
I'm wondering how `this` gets captured with empty `[]`.

Reply:
weird indeed

Reply:
Is threadTracker_ a `static`?

Reply:
@Dr15Jones pointed out that `threadTracker_` is static.

Reply:
Thanks